repository/rest-proxy/operator/params.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: kudo.dev/v1beta1
+parameters:
+  - name: HOST_NAME
+    description: "Host name used to generate absolute URLs in responses"
+    default: "localhost"
+  - name: SCHEMA_REGISTRY_URL
+    description: "The base URL for Confluent Schema Registry that should be used by the Avro serializer"
+    default: "http://localhost:8081/"
```

Comment:
let's use what would be default schema registry URL of the schema registry operator

Reply:
with this I mean, if a user installs zookeeper/kafka/schema-registry/rest-proxy in the same namespace, it should work with the default parameters

Reply:
Done. Changed as suggested.